gem5-configs/system/__init__.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# Copyright (c) 2016 Jason Lowe-Power
+# Copyright (c) 2020 The Regents of the University of California.
```

Comment:
This is fine, since I'm approving it, and I'm "Jason Lowe-Power" :).

However, generally, you can't "remove" anyone's copyright. Instead of removing, we should *add* another copyright line (if needed).

Reply:
I am sorry, I don't know much about copyrights, I have pushed these changes to Gerrit, should I resubmit taking this into account?

Reply:
You should probably revert this change on gerrit. If you make significant changes to a file, you should update the copyright to add your own (or UC Regents, in this case).